zingo-testutils/src/chain_generic_tests.rs

```diff
@@ -72,11 +72,11 @@ pub mod fixtures {
     use crate::chain_generic_tests::conduct_chain::ConductChain;
 
     /// runs a send-to-receiver and receives it in a chain-generic context
-    pub async fn propose_and_broadcast_value_to_pool<TE>(send_value: u64, pooltype: PoolType)
+    pub async fn propose_and_broadcast_value_to_pool<CC>(send_value: u64, pooltype: PoolType)
     where
-        TE: ConductChain,
+        CC: ConductChain,
```

Comment:
I thought a `ConductChain` was a trait?   Not a generic.  Every implementer of ConductChain..  that I am aware of, is a TestEnvironment....